assets/wallet/hot_wallet.json

```diff
@@ -0,0 +1,28 @@
+{
+	"metadata": {
+		"label": "hot_wallet",
+		"category": "wallet",
+		"subcategory": "",
+		"website": "https://hot-labs.org",
+		"description": "",
+		"organization": "hot_labs"
+	},
+	"addresses": [
+		{
+			"address": "EQBxrkqb9sVVGBVqNJzJW9lDcKwhhgeamkBJNt1njgo_tU7s",
+			"source": "",
+			"comment": "Ton.Fun referral address",
+			"tags": ["defi", "referral-address"],
+			"submittedBy": "ohld",
+			"submissionTimestamp": "2025-01-01T00:00:01Z"
+		},
+		{
+			"address": "EQCuVv07tBHuJrgFrMcDJFHESoE6TpLLoNTuqdL2LkXi7JGM",
+			"source": "",
+			"comment": "bridge inside hot_wallet",
```

Comment:
may be we should add it to the bridge category, since we already have a bridge dashboard based on its labels